flaxandteal/coral-arches
# Feat (#2144): Add IHR Number Validation

### Description
This adds validation to the IHR input field.
It will check that the string has the format 00000:000:00 and if not it will display an error message
It will also check for any existing entries in the database and throw an error if true

### Test
- Register the IHR Validation Function
- Reload the monument model
- Start an add IHR workflow
- Save and Continue to the Asset Details page
- Enter a random string eg. 'test'
- Should receive a warning error that the incorrect format is being used for the IHR
- Enter the correct format eg. 12345:123:12
- Should save successfully and show in the model resource
- After saving, start a new add ihr
- Enter the same number you previously entered
- Should throw an error that the IHR number has already been used
